confetti.js

```diff
@@ -39,7 +39,7 @@ class Confetti extends Component {
            duration: duration + this.randomIntValue(duration * .2, duration * -.2),
            toValue: windowHeight + 1.25,
            useNativeDriver: true
-        }).start(this.props.onComplete);
+        }).start(this.props.onAnimationComplete);
```

Comment:
For some reason, this callback was not working until I renamed it. I am guessing that it was conflicting with something up the hierarchy.